Source/DafnyCore/Resolver/DetectUnsoundFunctionReferences_Visitor.cs

```diff
@@ -0,0 +1,37 @@
+using System.Diagnostics.Contracts;
+
+namespace Microsoft.Dafny;
+
+class DetectUnsoundFunctionReferences_Visitor : ResolverBottomUpVisitor {
+  private readonly ICallable context;
+  public bool DoDecreasesChecks;
```

Comment:
I know it doesn't matter much, but I think C# developers generally prefer auto properties over public fields, at least I do. One example: you can put a breakpoint on a property getter/setter, but not on a field read/write.

Reply:
I made it private, as per the suggestion below.